test/e2e/apps/daemon_restart.go

```diff
@@ -329,7 +329,7 @@ var _ = SIGDescribe("DaemonRestart [Disruptive]", func() {
 		}
 	})
 
-	ginkgo.It("Kube-proxy should recover after being killed accidentally", func() {
+	ginkgo.It("kube-proxy should recover after being killed accidentally", func() {
```

Comment:
since other tests

```
test/e2e_node/gke_environment_test.go:  ginkgo.It("The iptable rules should work (required by kube-proxy)", func() {
```
 don't capitalize kube-proxy, making them uniform

Reply:
yeah, I agree that these ones that are really kube-proxy specific should be tagged